### PR TITLE
make better docs

### PR DESCRIPTION
**Pull request by juliettelavoie**
### Pull Request Checklist:
- [x] `pre-commit` hooks are installed/active in my local clone (`$ pre-commit install`)
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [ ] If a merge request has been made in parallel to this PR in xscen-notebooks, it is merged and the submodules have been updated.
- [ ] HISTORY.rst has been updated (with summary of main changes)
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Adding a need help? section to point people to the right place to ask questions
* Adding See Also section to point to xclim docs. Now, the links are not working, I think xclim needs to activate something, @Zeitsperre ? EDIT: It works.

Questions:
- Do we add see also links non-xclim functions? xesmf api section of the docs is empty... (https://xesmf.readthedocs.io/en/latest/user_api.html?highlight=regridder#regridder)
- In api.rst, the section Controlled Vocabulary and Mappings shows nothing. Do we want it to show something or delete it ?

### Does this PR introduce a breaking change?

no

### CODE REVIEW

**Comment by aulemahal** [comment]
@juliettelavoie  I added the missing config to point to xclim's doc. Tell me if it worked!

**Comment by juliettelavoie** [comment]
It works with scipy, but not with xclim... I remember Trevor saying something about activation in xclim.

**Comment by juliettelavoie** [comment]
Nevermind, it works with `xclim.core.calendar.convert_calendar`! There is an issue with the other ones, but it probably on my side.

File: xscen/config.py [hunk 1]
```diff
@@ -6,15 +6,12 @@
...
-.. code-block:: json
-
-    {
-        'module': {
-            'function' : {
-                ... kwargs ...
-             }
-        }
-    }
+.. code-block::
```
Comment (aulemahal) [inline xscen/config.py:9]:
```suggestion
.. code-block:: yaml
```
Does this exist?

**Review by aulemahal — APPROVED** [review]
Suggestion têteuse et absolument optionnelle, mais sinon, ça me va!

File: xscen/config.py (hunk 1 above)
Comment (Zeitsperre) [reply xscen/config.py:9]: It should be: https://pygments.org/docs/lexers/#pygments.lexers.data.YamlLexer

Comment (juliettelavoie) [reply xscen/config.py]: ça existe et c'est ajouté!

File: docs/index.rst [hunk 2]
```diff
@@ -4,7 +4,12 @@ Welcome to xscen's documentation!
 
 xscen: A climate change scenario-building analysis framework, built with Intake-esm catalogs and xarray-based packages such as xclim and xESMF.
 
-The documentation hosted in this repository is only partial. For the full documentation, please consult: https://scenario.gitlab.ouranos.ca/scenarios_main/
+Need help?
+==========
+* Ouranos employees can ask questions on the Ouranos private StackOverflow where you can tag subjects and people. (https://stackoverflow.com/c/ouranos/questions ).
+* Potential bugs in xscen can be reported as an issue here: https://github.com/Ouranosinc/xscen/issues .
+* Problem with data on Ouranos servers can be reported as an issue here: https://github.com/Ouranosinc/miranda/issues
```
Comment (RondeauG) [inline docs/index.rst:11]:
```suggestion
* Problems with data on Ouranos' servers can be reported as an issue here: https://github.com/Ouranosinc/miranda/issues
```

File: xscen/ensembles.py [hunk 3]
```diff
@@ -48,6 +48,11 @@ def ensemble_stats(
     -------
     xr.Dataset
         Dataset with ensemble statistics
+
+    See Also
+    --------
+    xclim.ensembles._base.create_ensemble, xclim.ensembles._base.ensemble_percentiles , xclim.ensembles._base.ensemble_mean_std_max_min
```
Comment (RondeauG) [inline xscen/ensembles.py:54]: These are not yet supported, but my PR should fix that.
```suggestion
    xclim.ensembles._base.create_ensemble, xclim.ensembles._base.ensemble_percentiles, xclim.ensembles._base.ensemble_mean_std_max_min, xclim.ensembles._robustness.change_significance, xclim.ensembles._robustness.robustness_coefficient, 
```

**Comment by RondeauG** [comment]
I think xESMF should absolutely be there for `regrid` and `aggregate.spatial_mean`, but as Juliette noted, their API appears to be completely broken. Suggestions?

Also, do we want to link to more basic stuff such as `mean()` and `interp()` for `aggregate.spatial_mean`?

**Review by RondeauG — APPROVED** [review]

**Comment by juliettelavoie** [comment]
Je pense que la question a peut-être été oublié alors je vais la répeter:
In api.rst, the section Controlled Vocabulary and Mappings shows nothing. Do we want it to show something or delete it ?

**Comment by RondeauG** [comment]
> Je pense que la question a peut-être été oublié alors je vais la répeter: In api.rst, the section Controlled Vocabulary and Mappings shows nothing. Do we want it to show something or delete it ?

Je crois que c'est un bug. `.. automodule:: xscen.CV`, mais on n'importe pas `CV` dans `__init__`. Je crois que ça devrais être `.. automodule:: xscen.utils.CV` ?

**Comment by juliettelavoie** [comment]
J'ai mis les See Also de xesmf.. En ce moment, ils ne pointent vers rien, mais au moins les gens peuvent voir que c'est basé sur autre chose et peuvent le googler.

**Comment by juliettelavoie** [comment]
> 

Ok, mais fait juste écrire le texte de description... 

> Mappings of (controlled) vocabulary. This module is generated automatically from json files in xscen/CVs. Functions are essentially mappings, most of which are meant to provide translations between columns.
> 
> Json files must be shallow dictionaries to be supported. If the json file contains a is_regex: True entry, then the keys are automatically translated as regex patterns and the function returns the value of the first key that matches the pattern. Otherwise the function essentially acts like a normal dictionary. The ‘raw’ data parsed from the json file is added in the dict attribute of the function.

Voulons-nous vraiment que ça apparaisse dans les docs?

**Comment by aulemahal** [comment]
Ah ça serait une bonne idée de montrer le contenu des jsons!

Sinon, je pense que le texte est quand même un peu pertinent, ne serait-ce que pour la mention du `.dict`.

**Comment by juliettelavoie** [comment]
Bon, j'ai ajouté les jsons. Je ne suis pas sur que c'était la meilleur façon de les inclure, mais je ne sais pas comment faire autrement. Ça ressemble à ça: 
Capture d’écran, le 2022-09-08 à 13 46 42

File: xscen/utils.py [hunk 4]
```diff
@@ -211,15 +211,52 @@ def maybe_unstack(
...
+        .. literalinclude:: ../xscen/CVs/xrfreq_to_frequency.json
+           :language: json
+           :caption: xrfreq_to_frequency
+
+        .. literalinclude:: ../xscen/CVs/xrfreq_to_timedelta.json
+           :language: json
+           :caption: xrfreq_to_timedelta
+
+
+        """
```
Comment (Zeitsperre) [inline xscen/utils.py:259]: Nice!

**Comment by aulemahal** [comment]
Ça me va comme ça!

**Pull request merged**